### DISCUSSION TITLE
How does it work?

### DISCUSSION THREAD

**Discussion by txl263** [sravioli/wezterm#21] [Q&A]
How does it work? I'm not familiar with Lua, and the configuration download has no effect, and I don't know how to start.

**Comment by mbaneshi** [comment]
I have been working with it for a long time. please follow the readme, and pay attention to prerequisites, it should work.

**Comment by sravioli** [comment]
> How does it work? I'm not familiar with Lua, and the configuration download has no effect, and I don't know how to start.

Hi, did you clone it in the correct folder? (inside `$HOME/.config/`).

When opening wezterm and pressing `CTRL+SHIFT+L` what do you see?
